### ISSUE TITLE
Error using nuget

### ISSUE THREAD

**Issue by Pietercal1** [nozzlegear/ShopifySharp#886]
Hi, I get the following error with nuget and 6.0.1.
It does not happen with the previous packages.
Thanks!
-------------------------------------------------------
NuGet\Install-Package : Found invalid data while decoding.
At line:1 char:1
+ NuGet\Install-Package ShopifySharp -Version 6.0.1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageComma 
   nd

**Comment by nozzlegear** [comment]
Hey! I'm honestly not sure what the problem might be here, I've never run into this issue before. Can you try running the install command using powershell + dotnet cli and see if you get a different result?

```ps
dotnet add package shopifysharp --version 6.0.1
```

**Comment by Pietercal1** [comment]
Hi,
Thanks for the reply!

It gave the same error...
HOWEVER, I saw that there was a version 6.0.1-b91 uploaded about an hour ago and that one installed just fine.
Is that a valid one that I can use?

Thanks!

**Comment by nozzlegear** [comment]
@Pietercal1 Sorry, I missed your last update. That 6.0.1-b91 version is a prerelease build – it *probably* works, but I can't guarantee it. The prerelease builds on nuget are published automatically whenever changes are pushed to the master branch. If those changes pass the automated integration tests we've set up, they'll get published and made available as a prerelease.

That said, I just published version 6.1.0 yesterday. Try that and let me know if you have any issues with it!
